### PR TITLE
AbsComponent H2

### PR DESCRIPTION
**Pull request by ntejos**
A patch for AbsComponents to work (at a basic level without checks) with H2 lines. This PR is only  meant to make pyigm's IGMguesses to be able to handle H2 molecules. A more fundamental solution is beyond the scope of this PR.

### CODE REVIEW

File: linetools/isgm/abscomponent.py [hunk 1]
```diff
@@ -190,14 +190,16 @@ def __init__(self, radec, Zion, zcomp, vlim, Ej=0./u.cm, A=None,
             self.sig_logN = 0.
 
         # Name
-        if name is None:
+        if (name is None) and (self.Zion != (-1, -1)):
```
Comment (profxj) [inline linetools/isgm/abscomponent.py]: Can you add to the doc string that -1,-1 is now special?

File: linetools/isgm/utils.py [hunk 2]
```diff
@@ -325,6 +326,55 @@ def complist_from_table(table):
     return complist
 
 
+def table_from_complist(complist):
+    """
+    Returns a QTable from an input list of AbsComponents. It only
```
Comment (profxj) [inline linetools/isgm/utils.py]: I have to admit that I'm no longer too enamored with QTable's.  I was at the start but find that normal Table objects do 99% of what is needed.

Do we need one here?

File: linetools/isgm/utils.py [hunk 3]
```diff
@@ -325,6 +326,55 @@ def complist_from_table(table):
...
+    fills in mandatory and special attributes (see notes below).
+    Information stored in dictionary AbsComp.attrib is ignored.
+
+    Parameters
+    ----------
+    complist : list of AbsCOmponents
```
Comment (profxj) [inline linetools/isgm/utils.py]: AbsComponents

File: linetools/isgm/abscomponent.py (hunk 1 above)
Comment (ntejos) [inline linetools/isgm/abscomponent.py]: Sure thing.

File: linetools/isgm/utils.py (hunk 2 above)
Comment (ntejos) [inline linetools/isgm/utils.py]: You are right; I find particularly annoying that vstack does not work properly with QTables. Will use Tables as default then. Still, there will be one place where QTable is more useful (e.g. to access entry values as quantity objects), but will be handled internally in the corresponding function; the user will always get a Table object out.

File: linetools/isgm/utils.py (hunk 3 above)
Comment (ntejos) [inline linetools/isgm/utils.py]: thanks.

**Comment by profxj** [comment]
Failing Python 3 tests.  Fix and then merge.

**Pull request merged**